apps/betterangels-backend/clients/enums.py

```diff
@@ -3,6 +3,14 @@
 from django.utils.translation import gettext_lazy as _
```

Comment:
where in the PRD I need to see like of accommodation OTHER needs to have accommodationOther or not ?

Reply:
it's in the ticket https://betterangels.atlassian.net/browse/DEV-714

> “other” does not include a custom text input